### ISSUE TITLE
Personal suggestion: Scroll set to none by default

### ISSUE THREAD

**Issue by ZYinMD** [lucax88x/CodeAceJumper#244]
I noticed there's a new setting `Scroll: 'none' | 'center' | 'top'`, and the default setting is `center`.

I personally think "none" should be the default setting, because I think it's the most natural. When you do the jump, you always focus your eyes on the target. When you complete the jump, the cursor lands on the target, and that's it, very natural. One additional scroll makes your eyes lose the target.

**Comment by lucax88x** [comment]
Yeah, I actually think the same.... I had the same feeling after I started using it, I'll make none default in the next days, thanks!

**Comment by liuhantang** [comment]
agree, default to 'center' gave me some surprise. For anyone who wants to set scroll mode to none, put this line in your vscode's `setting.json`
```
"aceJump.scroll.mode": "none"
```
